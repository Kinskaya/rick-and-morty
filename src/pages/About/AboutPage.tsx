import * as React from 'react';

class AboutPage extends React.Component {
  render() {
    return (
      <div>
        <h1>About us</h1>
      </div>
    );
  }
}

export default AboutPage;
