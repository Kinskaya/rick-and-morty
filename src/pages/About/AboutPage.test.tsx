import { render } from '@testing-library/react';
import AboutPage from './AboutPage';

describe('AboutPage component', () => {
  it('render correctly', () => {
    const { asFragment } = render(<AboutPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
