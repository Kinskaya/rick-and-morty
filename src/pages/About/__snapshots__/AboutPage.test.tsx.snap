// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage component render correctly 1`] = `
<DocumentFragment>
  <div>
    <h1>
      About us
    </h1>
  </div>
</DocumentFragment>
`;
